src/NewCommand.php

```diff
@@ -170,13 +173,14 @@ protected function defaultBranch()
      * @param  \Symfony\Component\Console\Output\OutputInterface  $output
      * @return void
      */
-    protected function installJetstream(string $directory, string $stack, bool $teams, InputInterface $input, OutputInterface $output)
+    protected function installJetstream(string $directory, string $stack, bool $teams, string $testSuite, InputInterface $input, OutputInterface $output)
```

Comment:
No need to pass this through here. Check the `--pest` option inside this method.

Reply:
Ok, I will push the new changes.

Reply:
Wouldn't it be better to ask whether users want to choose Pest or not? if I move the option check inside the installJetstream method, we shouldn't ask a question anymore.
the reason that I checked the option in there was to ask all the questions on the first of install.

Reply:
They can choose Pest by providing `--pest`. This makes the command equal to how Jetstream install works. There's no need for a question.

Reply:
Done.